### ISSUE TITLE
nan values in the pulse shape parameters

### ISSUE THREAD

**Issue by crhuffer** [crhuffer/LifetimeAnalysis#3]
I need to verify that none of the pulse shape parameters are returning nan values. I also need to write some test code to verify that none of the values are nan before the files are saved to the hard drive.

**Comment by crhuffer** [comment]
There are a couple of different waves that could have nan values, but the ones that I care about at the moment are the main pulse shape parameters area, height, and kurtosis. We do have some nan values still in the analysis. I have gone through a few of them by hand and have found that there isn't a peak in the region that we are looking for one.  In the first 5 such events for s16r4b1m1.dat, these all seem to be random afterpulsing from a large event. Therefore, we don't really want to include these events anyway.

**Comment by crhuffer** [comment]
We should consider how we want to deal with these events. It seems reasonable that we would throw them out. It is probably okay to leave them assuming that they are a very small fraction of all events.  If we assume that these events are solely caused by large muon events and that the large muon events are constant in time then it shouldn't include a systematic effect if we leave these events in the analysis.

**Comment by crhuffer** [comment]
I think we came to the conclusion that we know where these events are coming from, afterpulsing after very large events. At this point I have updated the code to track when we do not find a peak in the main detection channels and to store it in PNTerror. I have also updated the mid level analysis to remove any events from the timestamp wave (by naning them) if pnterror > 0. This removes all of the events where not peak is found in the main detection channels.  Therefore, although I never addressed this issue directly in the commits, it has been addressed. I will therefore close it out.
